examples/geo.html

```diff
@@ -0,0 +1,108 @@
+<html>
```

Comment:
Instead of adding an example, it would be better to add a test PDF and unit test for this new API. See `api_spec.js`.